proprietary/tokens/src/components/amsterdam/ordered-list.tokens.json

```diff
@@ -14,13 +14,26 @@
         "font-size": { "value": "{amsterdam.typography.text-level.6.wide.font-size}" }
       },
       "item": {
-        "margin-inline-start": { "value": "2.5rem" },
-        "padding-inline-start": { "value": "0" }
+        "margin-inline-start": {
+          "value": "2.25rem",
+          "comment": "Divide your total indentation width over margin and padding to position the marker."
+        },
+        "padding-inline-start": {
+          "value": "0.25rem",
+          "comment": "The total level 1 indentation for Amsterdam is 40 pixels, or 2.5rem."
+        }
       },
       "ordered-list": {
         "list-style-type": { "value": "lower-alpha" },
         "item": {
-          "margin-inline-start": { "value": "1.5rem" }
+          "margin-inline-start": {
+            "value": "1.5rem",
+            "comment": "Indent less than the parent to start-align the child’s marker with the parent text."
```

Comment:
I think start-aligning the child's marker to the parent isn't really possible with static values, because we have fluid font-sizes. We could use a fluid value here, but I'm also fine with it not being perfectly aligned.

Reply:
Wait yes, I thought I covered that as I considered rems to be dynamic, but they aren’t of course. We could do the clamping trick there, but the misalignment on narrow screens does seem very minimal.

And hopefully we can clamp the font-size on the root element, so that 1rem becomes viewpport-dependent.

So I’d propose leaving this as-is.

Reply:
> And hopefully we can clamp the font-size on the root element, so that 1rem becomes viewpport-dependent.

I think this would overwrite the user preference for the root font-size, no? We should be careful with that. 

Leaving this as-is for now is fine by me!